muffnn/mlp/base.py

```diff
@@ -252,6 +252,10 @@ def _set_up_graph(self):
                 if self.keep_prob != 1.0:
                     t = tf.nn.dropout(t, keep_prob=self._keep_prob)
                 t = affine(t, layer_sz, scope='layer_%d' % i)
+
+            if self.transform_layer_index == i:
+                self.transform_layer_ = t
```

Comment:
This should be `self._transform_layer`. Also, the way this is coded you would use the first hidden layer as the features by default which seems a bit strange.

Reply:
What would you prefer default to be? I chose it b/c you usually have at least one hidden layer so it will give something you wouldn't normally get with a predict function.

Reply:
The last hidden layer.

Reply:
ok